gradle.properties

```diff
@@ -3,11 +3,16 @@
 org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
 
-# Forge gradle
-minecraft_version=1.17.1
+# Minecraft and Forge
+minecraft_version=1.18
+forge_version=38.0.13

```

Comment:
Update to version 38.0.17 to prevent security issue and force a secure version on the user

Reply:
Shouldn't actually matter for mods. It only matters for for the installed version of the user. Still it doesn't hurt to update.

Reply:
yep, but if we require a newer version, user need to have the new version.
furthermore 38.0.14 had a PR we might need